### PR TITLE
Update version to 1.3.0 and add a release note for this version. (#156)

### PR DESCRIPTION
**Pull request by deepdatta**
Signed-off-by: Deep Datta <deedatta@amazon.com>

### Description
[Describe what this change achieves]

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has documentation added
- [X] Commits are signed as per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dlvenable — APPROVED** [review]
I tried to create a backport instead, but the infrastructure for it isn't working. This manual merge PR should be good for now.

I created #159 to support backports.

File: release-notes/logstash-output-opensearch.release-notes-1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Version 1.3.0 Release Notes
```
Comment (sshivanii) [inline release-notes/logstash-output-opensearch.release-notes-1.3.0.md:1]: Is the release note auto-generated, looks like it's not.

**Review by sshivanii — APPROVED** [review]
We can clean it up later, this PR is good to go

**Comment by deepdatta** [comment]
> I tried to create a backport instead, but the infrastructure for it isn't working. This manual merge PR should be good for now.
> 
> I created #159 to support backports.

Thanks David

**Comment by sshivanii** [comment]
Looks like DCO is failing even though you have signed the commit. You may want to verify the correct email is etup in your git config:
```
git config --global user.email "correct_email@example.com"
```

**Pull request merged**